malariagen_data/af1.py

```diff
@@ -272,61 +244,6 @@ def _transcript_to_gene_name(self, transcript):
 
         return parent_name
 
-    def genome_features(
-        self, region=None, attributes=("ID", "Parent", "Note", "description")
-    ):
-        """Access genome feature annotations.
-
-        Parameters
-        ----------
-        region: str or list of str or Region or list of Region
-            Contig name (e.g., "2RL"), gene name (e.g., "LOC125767311"), genomic
-            region defined with coordinates (e.g., "2RL:44,989,425-44,998,059") or a
-            named tuple with genomic location `Region(contig, start, end)`.
-            Multiple values can be provided as a list, in which case data will
-            be concatenated, e.g., ["2RL", "3RL"].
-        attributes : list of str, optional
-            Attribute keys to unpack into columns. Provide "*" to unpack all
-            attributes.
-
-        Returns
-        -------
-        df : pandas.DataFrame
-            A dataframe of genome annotations, one row per feature.
-
-        """
-
-        # Here we override the superclass implementation in order to provide a
-        # different default value for the `attributes` parameter, because the
-        # genome annotations don't include a "Name" attribute but do include a
-        # "Note" attribute which is probably useful to include instead.
-        #
-        # Also, we take the opportunity to customise the docstring to use
-        # examples specific to funestus.
-        #
-        # See also https://github.com/malariagen/malariagen-data-python/issues/306
-
-        return super().genome_features(region=region, attributes=attributes)
```

Comment:
Removed because this override is not needed any more, the default attributes are now handled differently.